app/uk/gov/hmrc/pillar2submissionapi/controllers/UKTaxReturnController.scala

```diff
@@ -24,23 +24,23 @@ import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.pillar2submissionapi.controllers.actions.{IdentifierAction, SubscriptionDataRetrievalAction}
 import uk.gov.hmrc.pillar2submissionapi.controllers.error.{EmptyRequestBody, InvalidJson}
 import uk.gov.hmrc.pillar2submissionapi.models.uktrsubmissions.UKTRSubmission
-import uk.gov.hmrc.pillar2submissionapi.services.SubmitUKTRService
+import uk.gov.hmrc.pillar2submissionapi.services.UKTaxReturnService
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 import uk.gov.hmrc.play.http.HeaderCarrierConverter
 
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class UKTRSubmissionController @Inject() (
+class UKTaxReturnController @Inject() (
   cc:                       ControllerComponents,
   identify:                 IdentifierAction,
   verifySubscriptionExists: SubscriptionDataRetrievalAction,
-  submitUKTRService:        SubmitUKTRService
+  ukTaxReturnService:       UKTaxReturnService
 )(implicit ec:              ExecutionContext)
     extends BackendController(cc) {
 
-  def submitUKTR: Action[AnyContent] = (identify andThen verifySubscriptionExists).async { request =>
+  def submitUktr: Action[AnyContent] = (identify andThen verifySubscriptionExists).async { request =>
```

Comment:
not sure why you changed this

Reply:
consistency, we were using a mix of all-caps and lower case method names, I have now changed them all to submitUKTR